test/versioning.test.js

```diff
@@ -49,15 +50,23 @@ describe('versioning', function() {
 
     it('should detect abi for custom node target', function() {
         var mock_process_versions = {
-            "node": '0.10.0',
-            "modules": '11',
+            "node": (jsEngine) ? '8.1.2' : '0.10.0',
```

Comment:
>jsEngine [](start = 21, length = 8)

People can build node-v8 from node-chakracore source in which case `process.jsEngine` will be `v8` . If we want strict chakracore check then do `jsEngine === 'chakracore'` instead.